maint_tools/check_pxd_in_installation.py

```diff
@@ -0,0 +1,72 @@
+"""Utility for testing presence and usability of .pxd files in the installation
+
+Usage:
+------
+python check_pxd_in_installation.py path/to/install_dir/of/scikit-learn
+"""
+
+import os
+import sys
+import glob
+import tempfile
+import textwrap
+import subprocess
+
+
+skl_dir = sys.argv[1]
+skl_dir = os.path.abspath(skl_dir)
+
+# find all .pxd files
+pxd_files = glob.glob(os.path.join(skl_dir, '**', '*.pxd'), recursive=True)
```

Comment:
By the way for this kind of helper maintenance script we could use recent feature of Python 3.6+ such as pathlib:

```python
import pathlib

tmpdir = pathlib.Path(tmpdir.name)
setup_path = tmpdir / "setup.py"

sklearn_dir = pathlib.Path(sys.argv[1])
pxd_files = sklearn_dir.glob("**/*.pxd")


print("> Found pxd files:")
for pxd_file in pxd_files:
    print(' -', str(pxd_file))

...
```